startup_scripts/220_prefixes.py

```diff
@@ -0,0 +1,60 @@
+from dcim.models import Site
+from ipam.models import Prefix, VLAN, Role, VRF
+from ipam.constants import PREFIX_STATUS_CHOICES
+from tenancy.models import Tenant, TenantGroup
+from extras.models import CustomField, CustomFieldValue
+from ruamel.yaml import YAML
+
+from netaddr import IPNetwork
+from pathlib import Path
+import sys
+
+file = Path('/opt/netbox/initializers/prefixes.yml')
+if not file.is_file():
+  sys.exit()
+
+with file.open('r') as stream:
+  yaml = YAML(typ='safe')
+  prefixes = yaml.load(stream)
+
+  optional_assocs = {
+    'site': (Site, 'name'),
+    'tenant': (Tenant, 'name'),
+    'tenant_group': (TenantGroup, 'name'),
+    'vlan': (VLAN, 'name'),
+    'role': (Role, 'name'),
+    'vrf': (VRF, 'name')
+  }
+
+  if prefixes is not None:
+    for params in prefixes:
+      custom_fields = params.pop('custom_fields', None)
+      params['prefix'] = IPNetwork(params['prefix'])
+
+      for assoc, details in optional_assocs.items():
+        if assoc in params:
+          model, field = details
+          query = { field: params.pop(assoc) }
+
+          params[assoc] = model.objects.get(**query)
+
+      if 'status' in params:
+        for prefix_status in PREFIX_STATUS_CHOICES:
+          if params['status'] in prefix_status:
+            params['status'] = prefix_status[0]
```

Comment:
Sure, I didn't add it because in the example you sent it wasn't done https://github.com/netbox-community/netbox-docker/blob/master/startup_scripts/110_devices.py#L52-L55 , since those lists are like 3 items each I thought you didn't want to use breaks. Now it's there.